client/homebrew/pages/editPage/editPage.jsx

```diff
@@ -424,6 +424,7 @@ const EditPage = createClass({
 						lang={this.state.brew.lang}
 						currentEditorPage={this.state.currentEditorPage}
 						allowPrint={true}
+						showHeaderNav={true}
```

Comment:
Does this component really need to stretch all the way to up to editPage.jsx and sharePage.jsx files?  It seems like we could just assume that the headerNav is always going to present in both pages (not always visible, of course), so we don't need to pass it in as a boolean prop on BrewRenderer-- it could just be included in BrewRenderer unconditionally.

Reply:
It could, if we also want it to appear on every other page, like New, Home, Change Log, Welcome...

Reply:
I don't really see a problem with that.  For something like the changelog, especially, it'd be nice to just have a list of dates/versions.  Same deal with FAQ (though it's obviously not very long at this point).  The point of the site is to create documents, and I'm not sure we need to say "some documents have these tools, others don't".  

If there are pages where navigation by header *isn't* desired, then we should maybe consider stop making everything out of special-case brews and just make normal pages.

Reply:
I agree with G-Ambatte here, FAQ perhaps, but the rest don't need navigation at all, 2 page documents....

Reply:
I’m not sure why there needs to be a distinction between short and long documents here. For all documents are we going to make the button disappear if it’s shorter than 2 pages? What if it is 10 pages but has no headers? 

The goal should be to have a consistent, unified UI as much as possible. If a button exists for one brew, it should be available on all brews. If, for whatever reason, a button really surely doesn’t do anything on a particular brew, then it should still be there but be disabled. That isn’t the case here, though.